Transformations/Transform/Transform.cpp

```diff
@@ -176,12 +177,45 @@ namespace Transform
 
         return rotmat;
     }
+
+    cv::Vec3d rotmat2euler(cv::Mat rotmat, std::string orientation = "zyx")
```

Comment:
Note! `orientation` has not been applied yet. It currently only works for zy'x''.